renderer/src/components/Totals/__tests__/Totals.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { render, screen, fireEvent, within } from "@testing-library/react";
+import Totals from "../Totals";
+import { globalIpcRendererMock } from "@/tests/mocks/electron";
+
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(() => ({ push: jest.fn() })),
+}));
+
+jest.mock("electron", () => ({
+  ipcRenderer: {
+    send: jest.fn(),
+    invoke: jest.fn(),
+    on: jest.fn(),
+    removeAllListeners: jest.fn(),
+    removeListener: jest.fn(),
+  },
+}));
+
+global.ipcRenderer = {
+  on: jest.fn(),
+  removeAllListeners: jest.fn(),
+  removeListener: jest.fn(),
+  send: jest.fn(),
+  invoke: jest.fn(),
+  ...globalIpcRendererMock,
+};
+
+const fakeDate = new Date("2024-03-20T11:30:00");
+const fakeTomorrowDate = new Date("2024-03-21T11:30:00");
+
+describe("Totals date changes", () => {
+  it("displays the correct date text for 20", () => {
```

Comment:
Please use this 
```
afterAll(() => {
    jest.restoreAllMocks();

    global.ipcRenderer = globalIpcRendererMock;
  });
```
An example you can see in this file: `renderer/src/components/GoogleConnection/__tests__/GoogleConnection.test.tsx`